src/routes/backpack/logmanager.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+|--------------------------------------------------------------------------
+| Backpack\LogManager Routes
+|--------------------------------------------------------------------------
+|
+| This file is where you may define or change all of the routes that are
+| handled by the Backpack\LogManager package.
+|
+*/
+
+Route::group([
+            'namespace'  => 'Backpack\LogManager\app\Http\Controllers',
```

Comment:
Ok, sometimes these are aligned and sometimes they are not; I prefer when they are but I think StyleCI is set to not align => and = (or maybe it's just personal preference).